packages/suite-base/package.json

```diff
@@ -82,7 +82,7 @@
     "@types/react-dom": "18.3.0",
     "@types/react-hover-observer": "workspace:*",
     "@types/react-transition-group": "^4.4.8",
-    "@types/react-virtualized": "9.21.29",
+    "@types/react-virtualized-auto-sizer": "1.0.4",
```

Comment:
It was not exactly replaced, It is the same library but isolated. It means it will load only the AutoSizer to the project instead of all react-virtualized and only use this component. You can check more info directly in the NPM:

https://www.npmjs.com/package/react-virtualized-auto-sizer